FAnsiSql/Implementations/PostgreSql/PostgreSqlServerHelper.cs

```diff
@@ -66,7 +66,7 @@ public override IEnumerable<string> ListDatabases(DbConnectionStringBuilder buil
         //create a copy so as not to corrupt the original
         var b = new NpgsqlConnectionStringBuilder(builder.ConnectionString)
         {
-            Database = null,
+            Database = "postgres", //Use the default database, otherwise Npgsql will try to connect to a database with the same name as the user
```

Comment:
We're actually best _deleting_ this line, rather than assigning any value to it.

Reply:
If we delete this line it will try to connect to a db with the same name as the user connecting. Is that the behaviour we want?

Reply:
No, it will connect to whatever db the user specified in the connection string, which is the only one we can reasonably rely on existing. We can't create databases at all on Oracle (it has a very "Oracle" approach to that!) and won't normally have the right permissions to do that on HIC or EPCC systems either, what matters is being able to connect reliably with the connection string provided. If this code fails with line 69 deleted, it means the user gave us an invalid connection string and it's obvious how to fix it.

Reply:
That makes sense, suppose the real issue is finding a clean way of dealing with the associated RDMP issue...